dependencies/models/yolo3_anchors.txt

```diff
@@ -0,0 +1 @@
+10,13,  16,30,  33,23,  30,61,  62,45,  59,119,  116,90,  156,198,  373,326
```

Comment:
Copied from keras-YOLOv3-model-set because it's needed for post-processing the model output.